README.md

```diff
@@ -126,13 +126,11 @@ brew install java automake libtool gmp mpfr pkg-config maven libffi llvm@14 open
 make libsecp256k1
 ```
 
-**NOTE**: Previous versions of these instructions required the user to use either the homebrew version of `flex` or the xcode command line tools version, with the wrong option giving an error.
+**NOTE**: Previous versions of these instructions required the user to use either the homebrew version of `flex` or the XCode command line tools version, with the wrong option giving an error.
 The current recommendation is to use the homebrew version.
```

Comment:
I think remove this note for now, just say "You should use the homebrew version of `flex` and XCode.